spec/system/composer/prosemirror_editor_spec.rb

```diff
@@ -1140,4 +1140,148 @@ def body(title)
       )
     end
   end
+
+  describe "image toolbar" do
+    it "shows image toolbar when image is selected" do
```

Comment:
IMO there is not much need for this spec because it's being tested indirectly by the others